projects/dendrites/permutedMNIST/experiments/centroid.py

```diff
@@ -145,10 +150,38 @@ class CentroidFigure1BExperiment(CentroidFigure1B,
     ),
 )
 
+# Synaptic Intelligence + Dendrites on 10 permutedMNIST tasks
+SI_CENTROID_10 = deepcopy(CENTROID_10)
+SI_CENTROID_10["model_args"].update(hidden_sizes=[2000, 2000])
```

Comment:
We have been using hidden layers with size `[2048,2048]`

Reply:
Both SI and SI+XdG use 2000 hidden layers per unit, that's why I changed it here, for a more fair comparison. However, our dendrite models alone (without SI) are still using 2048 hidden layers.

Reply:
Ok, we may want to normalize this so we have at least some datapoints where the feed forward portions are equal. We can also include high dimensional versions as long as it's explicit.

Reply:
if you have SI only and SI+dendrite on the graph I think its better to have 2048 to have everything with the same number of unit and make it more comparable with what we already have. We can always explicitly specify the difference in the text and if needs be show the 2000 unit SI only in a supplementary figure.

Reply:
It seems to me that perhaps we should just re-run all experiments with 2,000 hidden units, for both a) dendrites and b) SI + dendrites. That would be a better comparison, and I doubt the hyperparameters we found would change at all. Thoughts?